Modules/Tracker/LinePool.lua

```diff
@@ -218,128 +237,6 @@ function LinePool.Initialize(trackedQuestsFrame, UntrackQuest, TrackerUpdate)
     end
 end
 
----@param trackedAchievementsFrame Frame
-function LinePool.InitializeAchievementLines(trackedAchievementsFrame, OnAchievementClick, AchievementUpdate)
```

Comment:
I know see that it wasn't the best idea to separate the Achievement tracking from the Tracker the way I did it 😅 Lots of duplicated code 🙈